recipes-domd/domd-agl-cluster-demo-platform/domd-agl-cluster-demo-platform.bbappend

```diff
@@ -17,14 +17,18 @@ SRC_URI_rcar_append = " \
     repo://github.com/xen-troops/manifests;protocol=https;branch=master;manifest=prod_cockpit_demo_src/domd.xml;scmdata=keep \
 "
 
-XT_QUIRK_PATCH_SRC_URI_append_h3ulcb-4x2g-kf = "\
-    file://0001-linux-renesas-Remove-patch-230-from-renesas.scc.patch;patchdir=bsp/meta-rcar \
+XT_QUIRK_PATCH_SRC_URI_append = "\
```

Comment:
Well, this change is completely unrelated to the commit description.
Are you sure that commited the right changes?